### PR TITLE
Iteration 1

### PR DESCRIPTION
**Pull request by foleymichelle9**

### CODE REVIEW

File: test/team_test.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  end
+
+  def test_it_has_attributes
+    assert_equal 26, @team.team_id
+    assert_equal 14, @team.franchise_id
+    assert_equal 'FC Cincinnati', @team.team_name
+    assert_equal 'CIN', @team.abbreviation
+    assert_equal 'Nippert Stadium', @team.stadium
+    assert_equal '/api/v1/teams/26', @team.link
+
```
Comment (chrisstankus99) [inline test/team_test.rb:23]: Let's delete this space :)

File: lib/game_team.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    end
+
+    @@all = all_game_team
+  end
+
+  def self.all
+    @@all
+  end
+
+  attr_reader :game_id,
```
Comment (chrisstankus99) [inline lib/game_team.rb:21]: awesome job formatting these!

File: test/fixtures/game_teams_truncated.csv [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+2012030221,3,away,LOSS,OT,John Tortorella,2,8,44,8,3,0,44.8,17,7
+2012030221,6,home,WIN,OT,Claude Julien,3,12,51,6,4,1,55.2,4,5
+2012030222,3,away,LOSS,REG,John Tortorella,2,9,33,11,5,0,51.7,1,4
+2012030222,6,home,WIN,REG,Claude Julien,3,8,36,19,1,0,48.3,16,6
+2012030223,6,away,WIN,REG,Claude Julien,2,8,28,6,0,0,61.8,10,7
+2012030223,3,home,LOSS,REG,John Tortorella,1,6,37,2,2,0,38.2,7,9
+2012030224,6,away,WIN,OT,Claude Julien,3,10,24,8,4,2,53.7,8,6
+2012030224,3,home,LOSS,OT,John Tortorella,2,8,40,8,4,1,46.3,9,7
+2012030225,3,away,LOSS,REG,John Tortorella,1,7,25,13,2,1,50.9,5,3
+2012030225,6,home,WIN,REG,Claude Julien,3,8,35,11,3,1,49.1,12,9
```
Comment (kelshadarby) [inline test/fixtures/game_teams_truncated.csv:11]: Lets shorten this truncated file to here.

File: test/team_test.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
+require_relative 'test_helper'
+require './lib/team'
+
```
Comment (kelshadarby) [inline test/team_test.rb:3]: We can delete this extra space here. :)

File: lib/team.rb [hunk 5]
```diff
@@ -0,0 +1,37 @@
+require 'csv'
+
+class Team
+
+  @@all = []
```
Comment (kelshadarby) [inline lib/team.rb:5]: Nice setup!

File: lib/game.rb [hunk 6]
```diff
@@ -0,0 +1,44 @@
+require 'csv'
+
+class Game
+
+  @@all = []
+
+  def self.create_games(csv_file_path)
```
Comment (kelshadarby) [inline lib/game.rb:7]: I think we can create this in a module.

File: lib/game_team.rb (hunk 2 above)
Comment (foleymichelle9) [reply lib/game_team.rb:21]: Thanks :D

File: test/fixtures/game_teams_truncated.csv (hunk 3 above)
Comment (margoflewelling) [reply test/fixtures/game_teams_truncated.csv:11]: Yes, agreed! That will make calculating statistics much easier.

Comment (foleymichelle9) [reply test/fixtures/game_teams_truncated.csv:11]: good eye! 👀

**Pull request merged**